### PR TITLE
Completed Modal, List and typography

### PR DESCRIPTION
**Pull request by varun1233**
1. Modal Done
2. List Done
3. Typograhy done
4. Review Feedbacks
image
image
image

### CODE REVIEW

**Review by rahulk31 — APPROVED** [review]
Looks good to me Varun, Can be merged 👍

File: components/cards/cards.css [hunk 1]
```diff
@@ -31,7 +31,7 @@
 }
 .card-vertical:hover,
 .card-horizontal:hover {
-  box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px;
+  box-shadow: var(--bs-dark) 0px 5px 15px;
```
Comment (mehulsatardekar) [inline components/cards/cards.css]: use rems if it is more than 10px.

**Review by mehulsatardekar — APPROVED** [review]
Apart from that code looks good and ready to merge.

File: components/cards/cards.css [hunk 2]
```diff
@@ -5,7 +5,7 @@
   color: var(--primary-color);
 }
 .card-box-shadown {
-  box-shadow: rgba(149, 157, 165, 0.2) 0px 8px 24px;
+  box-shadow: var(--bs-light) 0px 8px 24px;
```
Comment (zoef-yt) [inline components/cards/cards.css]: use rem

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: components/cards/cards.css (hunk 2 above)
Comment (varun1233) [reply components/cards/cards.css]: Done

File: components/cards/cards.css (hunk 1 above)
Comment (varun1233) [reply components/cards/cards.css]: Done

**Pull request merged**